docs/reference/errors-and-warnings/NU3028.md

```diff
@@ -41,10 +43,13 @@ When the revocation check mode is set to `offline`, the warning will be downgrad
 <pre>The author primary signature's timestamp found a chain building issue: The revocation function was unable to check revocation because the certificate is not available in the cached certificate revocation list and NUGET_CERT_REVOCATION_MODE environment variable has been set to offline. For more information, visit https://aka.ms/certificateRevocationMode.</pre>
 
 #### Retry untrusted root failures
+
 > [!Note]
-> This option is available starting from NuGet 6.0.0 Preview 5 and only applies to the Windows-specific failure described in the [Issue](#issue) section.
+> This option is available starting from NuGet 6.0.0 Preview 5 and only applies to the Windows-specific failure described in the [Issue](#issue) section. The option does not apply to any other scenario and has no effect on Linux or macOS.
```

Comment:
Is it a good time to review the preview 5 mentioned? Downloading P5 is probably not even possible anymore.

```suggestion
> This option is available starting from NuGet 6.0.0 and only applies to the Windows-specific failure described in the [Issue](#issue) section. The option does not apply to any other scenario and has no effect on Linux or macOS.
```